packages/rest/src/manager.ts

```diff
@@ -338,10 +402,9 @@ export function createRestManager(options: CreateRestManagerOptions): RestManage
 
       const loggingHeaders = { ...payload.headers }
 
-      const authenticationScheme = payload.headers.authorization?.split(' ')[0]
-
       if (payload.headers.authorization) {
-        loggingHeaders.authorization = `${authenticationScheme} tokenhere`
+        const authorizationScheme = payload.headers.authorization?.split(' ')[0]
+        loggingHeaders.authorization = `${authorizationScheme} tokenhere`
```

Comment:
Is the tokenhere intentional?

Reply:
yes. thay will be logged with the depth set to1 and the log level to debug but it would be best to avoid printing the token for security